src/hevm/hevm-cli/hevm-cli.hs

```diff
@@ -298,6 +300,7 @@ unitTestOptions cmd testFile = do
     , EVM.UnitTest.askSmtIters = askSmtIterations cmd
     , EVM.UnitTest.smtTimeout = smttimeout cmd
     , EVM.UnitTest.solver = solver cmd
+    , EVM.UnitTest.covMatch = covMatch cmd >>= pure . pack
```

Comment:
Is there a more compact way to write this?

Reply:
```suggestion
    , EVM.UnitTest.covMatch = pack <$> covMatch cmd
```

Reply:
How about this?

Reply:
infix fmap, smart!